app/models/post.rb

```diff
@@ -1,3 +1,17 @@
 class Post < ApplicationRecord
   belongs_to :users
+  has_many :comments
+  has_many :likes
+
+  def five_most_recent_comments
+    comments.order(created_at: :desc).limit(5)
+  end
+
+  after_save :update_posts_counter
+
+  private
+
+  def update_posts_counter
+    users.increment!(:update_posts_counter)
```

Comment:
- [ ] After your refactor of the associations, what you will have is **`author`**, not `users`. Kindly change that.

- [ ] I think you meant to `increment!` the **`posts_counter`** column. Kindly do that.